Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,18 +67,58 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (ArrayIsNullOrEmpty(array))
+            {
+                return;
+            }
+
+            int[] arrayCopy = new int[array.Length];
+            CopyToArray(array, arrayCopy, 0, array.Length);
+            int j = 0;
+
+            for (int i = arrayCopy.Length - 1; i >= 0; i--)
+            {
+                array[j] = arrayCopy[i];
+                j++;
+            }
+        }
+
+        /// <summary>
+        /// Copy a section of an array in the begining of another array
+        /// </summary>
+        /// <param name="arraySource">Input array source.</param>
+        /// <param name="arrayDestination">destination array</param>
+        /// <param name="start">Starting index</param>
+        /// <param name="lenght">Lenght of new index</param>
+        private static void CopyToArray(int[] arraySource, int[] arrayDestination, int start, int lenght)
+        {
+            if (start < 0 || lenght > arraySource.Length)
+            {
+                return;
+            }
+
+            int j = 0;
+            for (int i = start; i < lenght; i++)
+            {
+                arrayDestination[j] = arraySource[i];
+                j++;
+            }
         }
 
+
         /// <summary>
         /// Remove last element in array.
         /// </summary>
         /// <param name="array">Input array.</param>
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
         {
-            // ToDo: implement.
-            return array;
+            if (ArrayIsNullOrEmpty(array))
+            {
+                return array;
+            }
+          
+            return RemoveAt(array, array.Length - 1); ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4930349)